build-system/global-configs/experiments-config.json

```diff
@@ -15,5 +15,12 @@
     "expirationDateUTC": "2019-10-10",
     "defineExperimentConstant": "_RTVEXP_INABOX_LITE"
   },
-  "experimentC": {}
+  "experimentC": {
```

Comment:
Should we revisit this keyed name approach in a future PR? There doesn't appear to be any value to this.

Reply:
These are tied to SCS's serving names, which only has A/B/C reserved.